docs/checkit.js

```diff
@@ -952,17 +952,19 @@ CheckIt.prototype.clearCalendarDiv = function () {
 };
 
 CheckIt.prototype.findCurrentDay = function () {
-    // Finds current day to apply CSS to it and so that the browser scrolls to it.
+    // Finds current day to apply CSS to it and so that the browser scrolls to // it.
     // Pass the day you are looking for as a parameter "YYYYMMDD"
-    // Clear the page of any days in the calendar that may have the class
-    // 'currentDay' on them
+    
+
     "use strict";
+    // if we don't have a calendar on the page for some reason, exit
     if (this.$calendarDiv === undefined) {
         return;
     }
-
-    if (this.$calendarDiv.find('.currentDay').length) {
-        this.removeClass("currentDay");
+    // Clear the page of any days in the calendar that may have the class
+    // 'currentDay' on them
+    if (this.$calendarDiv.find('#currentDay').length) {
```

Comment:
I'm confused about what you're trying to do here. Do you want just `this.$calendarDiv.find('#currentDay').removeAttr('id')` ?